aws-migration.rst

```diff
@@ -0,0 +1,30 @@
+.. post:: Jan 25, 2021
+   :tags: aws, downtime, infrastructure
+   :author: Eric
+   :location: BND
+
+
+Read the Docs Community downtime due to migrations to AWS
+=========================================================
+
+We wanted to make you aware that on Friday, February 12th at 1pm PST (4pm EST, 20:00 UTC),
+Read the Docs Community (readthedocs.org) will be having a **scheduled dashboard downtime of approximately 4 hours**.
+
+To ensure minimal impact for our users,
+we are performing this upgrade during a Friday afternoon
+which is one of our lowest usage periods.
+
+During this maintenance window, **documentation will continue to be online**
+but new documentation builds will not trigger and the Read the Docs dashboard will be read-only.
```

Comment:
```suggestion
but new documentation builds will not trigger and the Read the Docs dashboard will be unavailable.
```

Reply:
I thought we'd tested this with a read-only user in the DB? Seems pretty simple to apply while we're updating things, but yea it might break the dashboard. 🤷

Reply:
Ah yes, so internally we'll use a read-only connection to the database to keep docs up, but the dashboard will have to be completely down as it will throw exceptions everywhere with a read-only db connection